packages/components/src/DataTable/Body.tsx

```diff
@@ -5,10 +5,10 @@ import styles from "./DataTable.css";
 import { BodyLoading } from "./BodyLoading";
 
 interface BodyProps<T> {
-  table: Table<T>;
-  onRowClick?: (row: Row<T>) => void;
-  emptyState?: ReactNode | ReactNode[];
-  loading: boolean;
+  readonly table: Table<T>;
+  readonly onRowClick?: (row: Row<T>) => void;
+  readonly emptyState?: ReactNode | ReactNode[];
+  readonly loading: boolean;
```

Comment:
FYI for some reason my local branch had a change in this file that accidentally bled into here. I cleaned up the change but this is being auto applied and I can't get rid of it 🤦